config/settings.py

```diff
@@ -16,6 +16,18 @@
 
 SECRET_KEY = b'\xf4cx\xae\x97P\xff*Kj\xe1\xf1\xa4\x1a\xad\xb7\xcb~\xe44PDq\xf4\x11q\xf0\xecp\xb9\xc3\x19'
 
+# mail related
+INFO_EMAIL = "hulpa.yura@gmail.com"
```

Comment:
yeap. I will add default email. We already have one related to radio domain. Forgot to to remove this one. Thanks!